tests/Rector/MethodCall/RenameMethodCallRector/Fixture/fixture3.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Tests\Rector\MethodCall\RenameMethodCallRector\Fixture;
+
+use Nette\Utils\Html;
+
+class SomeClass4
+{
+    private function createHtml()
+    {
+        $html = new Html();
+        $html->add('someContent');
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Rector\MethodCall\RenameMethodCallRector\Fixture;
+
+use Nette\Utils\Html;
+
+class SomeClass4
+{
+    private function createHtml()
+    {
```

Comment:
Could also add rename for `createHtml`? So we know the `ClassMethod` is untouched